### DISCUSSION TITLE
Is there a way to change the default setting directory?

### DISCUSSION THREAD

**Discussion by pax0707** [airdcpp/airdcpp-windows#101] [Q&A] [answered]
Is there a way to move settings away from the Documents directory to something like AppData?
The current setup does not play nice with one drive...

**Comment by Sopor** [comment, marked as answer]
You can always use the portable version (or convert the already installed version to portable).

Documents and AppData are both in your user profile, so why do you want to move it there?.

Check the `dcppboot.xml` for more information.

I have been using the portable version for 20 years 😋

**Reply by pax0707** [reply]
I've totally forgotten it has a portable version.

For reasons too complicated to explain I use OneDrive and sync the Documents folder. Recently I've noticed it using way too much CPU and it looks like it doesn't like AirDC++ hash files/log files and tries to constantly sync them while they are open.

The portable version is working great. Just put the settings in the settings folder.

**Reply by pax0707** [reply]
Definitely made an impact.
![image](https://github.com/airdcpp/airdcpp-windows/assets/3536230/1059d01e-3aac-4695-920a-112965c47471)
